### PR TITLE
doc Install Windows

### PR DESCRIPTION
**Pull request by fabiohcnobre**
doc Install Windows

### CODE REVIEW

File: docs/install/index.md [hunk 1]
```diff
@@ -80,6 +80,12 @@ In case your distro doesn't have the package manager option listed above, procee
 ### Windows
 
 TBD in the version v0.0.8.
+- • Rust installation: [rust-lang.org/tools/install](https://www.rust-lang.org/tools/install)
```
Comment (raphamorim) [inline docs/install/index.md:83]: Can you move this to info to https://raphamorim.io/rio/install/#building ?

(By adding a new section with Windows after MacOS?)

You can skip the rust installation information (https://raphamorim.io/rio/install/#build-from-the-source)

**Comment by raphamorim** [comment]
@fabiohcnobre can you also add a screenshot in the README.md like https://github.com/raphamorim/rio/blob/main/resources/demo-x11.png or https://github.com/raphamorim/rio/blob/main/resources/demo-selection.png

File: docs/install/index.md (hunk 1 above)
Comment (fabiohcnobre) [reply docs/install/index.md]: Thanks.

File: docs/install/index.md [hunk 2]
```diff
@@ -79,7 +79,8 @@ In case your distro doesn't have the package manager option listed above, procee
 
 ### Windows
 
-TBD in the version v0.0.8.
+TBD in the version v0.0.8. 
+
```
Comment (raphamorim) [inline docs/install/index.md:83]: Please remove the extra line

File: docs/install/index.md [hunk 3]
```diff
@@ -158,4 +159,11 @@ macOS:
 make release-macos
 {% endhighlight %}
 
+Windows:
+
+{% highlight bash %}
+cargo build
+cargo run
```
Comment (raphamorim) [inline docs/install/index.md:166]: Maybe we can say that an executable will be built.

so something like:

```
{% highlight bash %}
cargo build --release
{% endhighlight %}

After the command execution an executable will be created called Rio.exe inside of “target/release”

Optionally you can also build and run the terminal with “cargo run”.
```

Comment (fabiohcnobre) [reply docs/install/index.md:166]: I found one faster. But you're right I'll leave it as you put it. Thank you

**Review by raphamorim — APPROVED** [review]

**Comment by raphamorim** [comment]
Thank you @fabiohcnobre

**Pull request merged**